biom/table.py

```diff
@@ -1879,19 +1927,10 @@ def _data_equality(self, other):
         necessary before performing the final comparison.
```

Comment:
Are all these docstring comments ("Matrices are equal iff the following items are equal <etc>") still appropriate to this particular method?  I'm wondering if they would make more sense on `__eq__` or with some of them going onto `_data_equality_meta`.

Reply:
This particular method, `_data_equality` is called on `__eq__` and should not exhibit different semantics from prior versions. However, I agree that they could be shuffled to more precise locations -- thanks! I'll do that in a moment